.github/ISSUE_TEMPLATE/request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 'Suggest a feature'
+about: Suggest an idea to make the tool better!
+title: "[request]: describe request here"
+labels: enhancement
+assignees: ''
+
+---
+## Proposal
+**Use case. Why is this important?**
+
+*“Nice to have” is not a good use case. :)*
```

Comment:
Saying what is not a good case seems a bit negative … can we find a positive formulation of what we _do_ want, for example

```suggestion
<!-- Please describe how this enhancement would improve your user experience -->
```